crates/red_knot_python_semantic/resources/mdtest/comparison/tuples.md

```diff
@@ -89,17 +88,17 @@ However, `==` and `!=` are exceptions and can still provide definite results.
 a = (1, 2)
 b = (1, "hello")
 
-# TODO: should be Literal[False]
-reveal_type(a == b)  # revealed: @Todo
+# TODO: should be Literal[False], once type-checking for rich comparison operands is implemented
+reveal_type(a == b)  # revealed: bool
 
-# TODO: should be Literal[True]
-reveal_type(a != b)  # revealed: @Todo
+# TODO: should be Literal[True], once type-checking for rich comparison operands is implemented
+reveal_type(a != b)  # revealed: bool
 
 # TODO: should be Unknown and add more informative diagnostics
-reveal_type(a < b)  # revealed: @Todo
-reveal_type(a <= b)  # revealed: @Todo
-reveal_type(a > b)  # revealed: @Todo
-reveal_type(a >= b)  # revealed: @Todo
+reveal_type(a < b)  # revealed: bool
+reveal_type(a <= b)  # revealed: bool
+reveal_type(a > b)  # revealed: bool
+reveal_type(a >= b)  # revealed: bool
```

Comment:
And we can also error on these cases, once we implement ordering comparisons on mismatched literals to always be an error